docs/sources/articles/systemd.md

```diff
@@ -24,14 +24,38 @@ If you want Docker to start at boot, you should also:
 ## Custom Docker daemon options
 
 There are a number of ways to configure the daemon flags and environment variables
-for your Docker daemon. 
+for your Docker daemon.
 
 If the `docker.service` file is set to use an `EnvironmentFile`
 (often pointing to `/etc/sysconfig/docker`) then you can modify the
-referenced file.
+referenced file. To check if this is the case:
 
-Or, you may need to edit the `docker.service` file, which can be in
-`/usr/lib/systemd/system`, `/etc/systemd/service`, or `/lib/systemd/system`.
+    $ sudo systemctl show docker --property EnvironmentFile
+    EnvironmentFile=-/etc/sysconfig/docker
+
+Otherwise, you can see the location and content of the service files:
+
+    $ sudo systemctl cat docker
+    # /lib/systemd/system/docker.service
+    [Unit]
+    Description=Docker Application Container Engine
+    Documentation=http://docs.docker.com
```

Comment:
Oh! Not your fault, but that doesn't look right, that should be `https://`. Looks like a mistake here; https://github.com/docker/docker/blob/9944f8b95453e3d0726fc037aaed41a98e72e070/contrib/init/systemd/docker.service#L3